YAFC/Workspace/ProductionTable/ProductionTableFlatHierarchy.cs

```diff
@@ -82,12 +87,17 @@ public class FlatHierarchy<TRow, TGroup> where TRow : ModelObject<TGroup>, IGrou
             flatGroups.MoveListElementIndex(indexFrom, indexTo);
         }
 
-        //private readonly List<(Rect, SchemeColor)> listBackgrounds = new List<(Rect, SchemeColor)>();
         private readonly Stack<float> depthStart = new Stack<float>();
+
         private void SwapBgColor(ref SchemeColor color) {
-            color = color == SchemeColor.Background ? SchemeColor.PureBackground : SchemeColor.Background;
+            color = nextRowIsHighlighted ? nextRowBackgroundColor :
+                color == SchemeColor.Background ? SchemeColor.PureBackground : SchemeColor.Background;
         }
 
+        public bool nextRowIsHighlighted { get; private set; }
```

Comment:
Please document new properties. So at some point we'll get more and more documentation of our code :smile:

Reply:
Done.